docs/config.md

```diff
@@ -1,211 +1,466 @@
+   the call instead of having a screen with a single button on it. This is most useful if the Jitsi instance being used already
+   has a landing page for users to test audio and video before joining the call, otherwise users will automatically join the call.
+   For example:
+   ```json
+   {
+      "jitsi_widget": {
+         "skip_built_in_welcome_screen": true
+      }
+   }
+   ```
+3. `voip`: Optional configuration for various VoIP features. Currently can only contain a single `obey_asserted_identity` value to
+   send MSC3086-style asserted identity messages during VoIP calls in the room corresponding to the asserted identity. This *must*
+   only be set in trusted environments. The option defaults to `false`. For example:
+   ```json
+   {
+      "voip": {
+         "obey_asserted_identity": false
+      }
+   }
+   ```
+4. `widget_build_url`: Optional URL to have Element make a request to when a user presses the voice/video call buttons in the app,
+   if a call would normally be started by the action. The URL will be called with a `roomId` query parameter to identify the room
+   being called in. The URL must respond with a JSON object similar to the following:
+   ```json
+   {
+      "widget_id": "$arbitrary_string",
+      "widget": {
+         "creatorUserId": "@user:example.org",
+          "id": "$the_same_widget_id",
+          "type": "m.custom",
+          "waitForIframeLoad": true,
+          "name": "My Widget Name Here",
+          "avatar_url": "mxc://example.org/abc123",
+          "url": "https://example.org/widget.html",
+          "data": {
+             "title": "Subtitle goes here"
+          }
+      },
+      "layout": {
+         "container": "top",
+         "index": 0,
+         "width": 65,
+         "height": 50
+      }
+   }
+   ```
+   The `widget` is the `content` of a normal widget state event. The `layout` is the layout specifier for the widget being created,
+   as defined by the `io.element.widgets.layout` state event.
+5. `audio_stream_url`: Optional URL to pass to Jitsi to enable live streaming. This option is considered experimental and may be removed
+   at any time without notice.
+
+## Bug reporting
+
+If you run your own rageshake server to collect bug reports, the following options may be of interest:
+
+1. `bug_report_endpoint_url`: URL for where to submit rageshake logs to. Rageshakes include feedback submissions and bug reports. When
+   not present in the config, the app will disable all rageshake functionality. Set to `"https://element.io/bugreports/submit"` to submit
+   rageshakes to us, or use your own rageshake server.
+2. `uisi_autorageshake_app`: If a user has enabled the "automatically send debug logs on decryption errors" flag, this option will be sent
+   alongside the rageshake so the rageshake server can filter them by app name. By default, this will be `"element-web"`, as with any other
```

Comment:
```suggestion
   alongside the rageshake so the rageshake server can filter them by app name. By default, this will be `element-web`, as with any other
```